coderadar-rest/src/main/java/io/reflectoring/coderadar/rest/query/GetDeltaTreeForTwoCommitsController.java

```diff
@@ -21,11 +21,7 @@ public GetDeltaTreeForTwoCommitsController(GetDeltaTreeForTwoCommitsUseCase getD
     }
 
     @RequestMapping(method = {RequestMethod.POST, RequestMethod.GET}, path = "/projects/{projectId}/metricvalues/deltaTree", consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE)
-    public ResponseEntity getDeltaTreeForTwoCommits(@Validated @RequestBody GetDeltaTreeForTwoCommitsCommand command, @PathVariable("projectId") Long projectId){
-        try {
-            return new ResponseEntity<>(getDeltaTreeForTwoCommitsUseCase.get(command, projectId), HttpStatus.OK);
-        } catch (IllegalArgumentException e){
```

Comment:
This cannot happen anymore?

Reply:
No, it cannot. Only `EntityNotFoundException`s are thrown and those are caught in the ExceptionHandler.